django/yaamod/wsgi.py

```diff
@@ -7,10 +7,13 @@
 https://docs.djangoproject.com/en/2.2/howto/deployment/wsgi/
 """
 
+import sys
 import os
-
 from django.core.wsgi import get_wsgi_application
 
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'yaamod.settings')
+path = '/var/yaamod.co.il/VG/django'
+if path not in sys.path:
+        sys.path.append(path)
```

Comment:
too much whitespace.
also it should be yaamod, not VG (it should be changed on the production server too)